modyn/models/dlrm/dlrm.py

```diff
@@ -132,7 +132,7 @@ def embedding(self) -> Optional[torch.Tensor]:
         return self.top_model.embedding
 
     @property
-    def embedding_recorder(self) -> EmbeddingRecorder:
+    def embedding_recorder(self) -> EmbeddingRecorder:  # type: ignore
```

Comment:
iirc. we are returning a read-only member in a non read-only property function.

Reply:
hm, I guess I don't fully get the problem here, but works for now i guess

Reply:
these property functions also support write access when returning member variables. e.g. `obj. embedding_recorder = some_other_embedding_recorder`. The nested object we are returning here is read only though. It would be possible to fix that by using a private class member instead, but I thought that wouldn't be worth the effort.